mysql-test/t/audit_log_json.test

```diff
@@ -7,17 +7,41 @@ SET GLOBAL audit_log_flush=ON;
 --remove_file $MYSQLD_DATADIR/test_audit.log
 SET GLOBAL audit_log_flush=ON;
 
+--let $TEST_CONTROL_CHARS=1;
```

Comment:
Uppercase vars in MTR become environment vars too, lowercase ones don't. Since you don't need an env var here (e.g. for interacting with inline Perl), use lowercase. This comment on its own would be too minor to force a re-spin of this PR